docs/sources/get-started/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+description: Get started with the Writer's Toolkit
+menuTitle: Get started
+title: Get started with the Writer's Toolkit
+weight: 50
+aliases:
+  - /docs/writers-toolkit/get-started/
+---
+
+# Get started with the Writer's Toolkit
+
+This guide provides includes everything you need to complete your documentation project from start to finish.
+In short, it's a `tl;dr` on Writer’s Toolkit to quickly point you in the right direction.
```

Comment:
I would consider `tl:dr` to be jargon.  I'm not sure that it's globally understood (and even if it is, always spell out acronyms on first use).